master/server.go

```diff
@@ -166,6 +198,31 @@ func (s *Server) Start(ctx context.Context) (err error) {
 	if test.GlobalTestFlag {
 		return s.startForTest(ctx)
 	}
+
```

Comment:
ToDo : ServerMaster should update the its address in etcd , so the executor can get it

Reply:
It will be implemented in another PR.